### PR TITLE
changed: set default line color if line color could not be found

### PR DESCRIPTION
**Pull request by dschemp**
Hi,

I've quickly addressed the issue mentioned in #59. It appears as if the module for line colors is out-of-date, misssing somethings or has typos.
To future-proof against this and new lines popping up later I've added a check if a color could be found. If not, set the foreground color to white and background to a dark grey.

As I am not that much into JS and Node, I hope this is an appropriate solution.

Thanks 👍

### CODE REVIEW

**Comment by deg0nz** [comment]
I will look into until the end of the week

**Comment by deg0nz** [comment]
There's just one thing I'd like to have changed. Please see review.

We can wait and leave the PR open until your changes in `vbb-line-colors` are merged. Would be so kind and change update the `package.json` accordingly when it's ready?

And please also bump the MINOR version of this module as last commit of this PR. :)

**Review by deg0nz — CHANGES_REQUESTED** [review]
Please change the comparison to strict JS

File: node_helper.js [hunk 1]
```diff
@@ -138,12 +138,19 @@ module.exports = NodeHelper.create({
     }
 
     // Change default values if we changed them
-    if ("bg" in colors) {
-      properties.bgColor = colors.bg;
+    if (colors != undefined) {
```
[the remark this answers is not in the document]
Comment (deg0nz) [reply node_helper.js:141]: Aahh, and another thing.

The better version here would be:
``` JS 
if (typeof colors !== "undefined") {
```

It's better practice to make checks for `undefined` with the `typeof` operator. 
(Please see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof for reference)

[the remark this answers is not in the document]
Comment (dschemp) [reply node_helper.js]: Fixed it. Thanks for telling me.

File: package.json [hunk 2]
```diff
@@ -32,7 +32,7 @@
   "dependencies": {
     "hafas-client": "^5.8.0",
     "moment": "^2.27.0",
-    "vbb-line-colors": "^1.0.2",
+    "vbb-line-colors": "^1.0.3",
```
Comment (deg0nz) [inline package.json:35]: Could you please change vbb-line-colors to `1.0.4`?

After that, I will pull your changes :)

Comment (dschemp) [reply package.json]: Done.

**Comment by deg0nz** [comment]
Nice. Thank you for this contribution @dschemp :)

**Review by deg0nz — APPROVED** [review]

**Pull request merged**